frontend/src/components/chatInput/ChatInput.styled.ts

```diff
@@ -48,11 +51,19 @@ const MemeGenerateIcon = styled(BsImage)`
     width: 26px;
     height: 26px;
     fill: ${({ theme }) => theme.gray_300};
+
+    @media (width <=${({ theme }) => theme.media_md}) {
+        width: 20px;
+        height: 20px;
+    }
 `;
 
 export const Counter = styled.p<{ isError: boolean }>`
-    min-width: 75px;
+    min-width: 4rem;
+
     color: ${({ theme, isError }) => (isError ? theme.red : theme.white)};
```

Comment:
Additional parentheses aren't necessary.

```suggestion
    color: ${({ theme, isError }) => isError ? theme.red : theme.white};
```

Reply:
Ye, i fix my settings in IDE. thanks :)

Reply:
Ye, but prettier do it automattically.
i have "arrowParens": "avoid", but it's not work in that case, we must leave it like that.